build.gradle

```diff
@@ -20,7 +20,7 @@ publishing {
 }
 
 group = 'com.github.hmcts'
-version = '1.4.4'
+version = '1.4.5'
```

Comment:
I think this should be 2.0.0 due to the proposed changes

Reply:
Version updated :)